src/js/schemas/service-schema/Volumes.js

```diff
@@ -81,7 +91,12 @@ const Volumes = {
       }
     },
     externalVolumes: {
-      title: 'Network Volumes',
+      title: 'External Volumes',
+      description: (
+        <span>
+        Create a stateful, fault-tolerant application with external volumes. <a href="https://docs.mesosphere.com/1.7/usage/storage/external-storage/">Learn more about external volumes.</a>
```

Comment:
We need to indent this once further and add `target="_blank"` to the external link.